docs/platform/api/upgrade-to-linode-api-v4.md

```diff
@@ -0,0 +1,190 @@
+    | PUT  | update a resource.  |
+    | DELETE  | remove a resource from your account.  |
+
+* Previous versions of the API allowed users to submit request and method parameters via URL parameters; this is no longer supported. All information needed for a request must be included in the body of a POST or PUT request.
+
+* Error codes also follow standard RESTful format; the custom error codes from previous versions are no longer used or supported.
+
+* Batch requests are no longer supported. Many of the actions that previously required multiple requests–such as creating and booting a new Linode or creating and attaching a Block Storage Volume–can now be achieved in a single request.
+
+## Creating a Linode
+
+The process for creating a new Linode has been streamlined in the new API. Previously, setting up a new Linode required multiple requests for creating the Linode, deploying an image, and booting. All of these steps have been combined, allowing you to get a running Linode with a single request:
+
+    curl -X POST https://api.linode.com/v4/linode/instances \
+    -H "Authorization: Bearer $TOKEN" -H "Content-type: application/json" \
+    -d '{
+      "type": "g5-standard-2",
+      "region": "us-east",
+      "image": "linode/debian9",
+      "root_pass": "root_password",
+      "label": "prod-1"
+      }'
+
+## Examples
+
+This section presents examples of how to convert some of the most common tasks from version 3 of the API to version 4.
+
+### Linodes
+
+#### List All Linodes
+
+To view all Linodes on your account:
+
+**API v3**
+
+    curl https://api.linode.com/?api_key=sekrit&api_action=linode.list
+
+**API v4**
+
+  In the new API, this is accomplished with a GET request to the `/instances` endpoint:
+
+    curl https://api.linode.com/v4/linode/instances -H "Authorization: Bearer $token"
+
+#### Boot a Linode
+
+**API v3**
+
+    curl https://api.linode.com/?api_key=sekrit&api_action=linode.boot&linodeID=123456
+
+**API v4**
+
+    curl https://api.linode.com/v4/linode/instances/$linode_id/boot -H "Authorization: Bearer $token"
+
+### NodeBalancers
+
+In the new API, `datacenter` has been replaced with `region`. Requests must also be converted to RESTful format, so for example creating a new NodeBalancer uses a POST request to the `/nodebalancers` endpoint:
+
+**API v3**
+
+    curl https://api.linode.com/?api_key=sekrit&api_action=nodebalancer.create&DatacenterID=123456&Label=this-balancer
+
+**API v4**
+
+    curl -H "Content-Type: application/json" \
+    -H "Authorization: Bearer $token" \
+    -X POST -d '{
+        "region": "us-east",
+        "label": "nodebalancer_1"
+    }' \
+    https://api.linode.com/v4/nodebalancers
+
+### StackScripts
+
+StackScripts have been moved under the `/linode` endpoint. To list all available StackScripts:
+
+**API v3**
+
+    curl https://api.linode.com/?api_key=sekrit&api_action=stackscript.list
+
+**API v4**
+
+    curl https://api.linode.com/v4/linode/stackscripts
```

Comment:
You should include an example of deploying from stackscript.  I prefer to plug StackScriptID 10079 when possible.